src/styles/blocks/catalog.scss

```diff
@@ -17,6 +17,12 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: transform $card-element-transition, color $card-element-transition;
+
+    &:hover {
+      transform: scale(1.2);
+      color: #34568b;
```

Comment:
would be nice to create a variable for this color

Reply:
variable created